packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -307,76 +341,78 @@ export const prepareLocalOrchestrationAccountKit = (
          * @param {string} validatorAddress
          * @param {Amount<'nat'>} ertpAmount
          */
-        async delegate(validatorAddress, ertpAmount) {
+        delegate(validatorAddress, ertpAmount) {
           // TODO #9211 lookup denom from brand
           const amount = {
             amount: String(ertpAmount.value),
             denom: 'ubld',
           };
           const { account: lca } = this.state;
 
-          const results = E(lca).executeTx([
-            typedJson('/cosmos.staking.v1beta1.MsgDelegate', {
-              amount,
-              validatorAddress,
-              delegatorAddress: this.state.address.address,
-            }),
-          ]);
-
-          return when(watch(results, this.facets.extractFirstResultWatcher));
+          return watch(
+            E(lca).executeTx([
+              typedJson('/cosmos.staking.v1beta1.MsgDelegate', {
+                amount,
+                validatorAddress,
+                delegatorAddress: this.state.address.address,
+              }),
+            ]),
+            this.facets.extractFirstResultWatcher,
```

Comment:
Is including `Watcher` in the name worthwhile? It's starting to feel like `extractFirstResultWatcherBehaviorDefinitionCodeSpecification`.

Reply:
I agree its verbose. This seems to be the pattern adopted elsewhere, so I ran with it. Technically it's a "watcher handler"